stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/coil/StreamImageLoaderFactory.kt

```diff
@@ -22,43 +22,36 @@ import coil.ImageLoader
 import coil.ImageLoaderFactory
 import coil.decode.GifDecoder
 import coil.decode.ImageDecoderDecoder
-import coil.util.CoilUtils
-import okhttp3.Dispatcher
-import okhttp3.Interceptor
-import okhttp3.OkHttpClient
+import coil.decode.VideoFrameDecoder
+import coil.disk.DiskCache
+import coil.memory.MemoryCache
 
 private const val DEFAULT_MEMORY_PERCENTAGE = 0.25
+private const val DEFAULT_DISK_CACHE_PERCENTAGE = 0.02
+private const val DISK_CACHE_DIRECTORY = "image_cache"
```

Comment:
For here, I referenced the official guide - https://coil-kt.github.io/coil/image_loaders/#caching